src/components/vm/consoles/consoles.jsx

```diff
@@ -81,15 +81,35 @@ export class ConsoleCardState extends StateObject {
     }
 }
 
-export const ConsoleCard = ({ state, vm, config, onAddErrorNotification, isExpanded }) => {
+export const ConsoleCard = ({
+    state,
+    vm,
+    config,
+    onAddErrorNotification,
+    isExpanded = false,
+    isStandalone = false,
+}) => {
     const serials = vm.displays.filter(display => display.type == 'pty');
     const inactive_serials = vm.inactiveXML.displays.filter(display => display.type == 'pty');
     const vnc = vm.displays.find(display => display.type == 'vnc');
     const inactive_vnc = vm.inactiveXML.displays.find(display => display.type == 'vnc');
     const spice = vm.displays.find(display => display.type == 'spice');
     const inactive_spice = vm.inactiveXML.displays.find(display => display.type == 'spice');
 
-    let type = state.type;
+    const lastVncRemoteSize = useRef([1024, 768]);
+
+    useEffect(() => {
+        if (isStandalone) {
+            const watchdog = cockpit.channel({ payload: "null" });
+            watchdog.addEventListener("close", () => {
+                console.debug("Closing detached VNC");
+                window.close();
+            });
+            return () => watchdog.close();
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2254-7fcc4fae-20250807-092345-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)